src/parse/ida.py

```
@@ -0,0 +1,38 @@
+import pandas as pd
+import numpy as np
```

Flake8 (7.2.0)

[error] 'numpy as np' imported but unused

(F401)